buildconfig/pygame-stubs/camera.pyi

```diff
@@ -8,7 +8,7 @@ def list_cameras() -> List[str]: ...
 class Camera:
     def __init__(
         self,
-        device: Optional[int] = 0,
+        device: str,
```

Comment:
I guess other implementations use int? Can this be `Optional[Union[int, str]]` or does this require string here?

Reply:
Think this was just an error. Two code paths are here:

https://github.com/pygame/pygame/blob/16a8d3ea36ff09fc1c1d91c6492219a4bef944db/src_c/_camera.c#L1787-L1800

and here:

https://github.com/pygame/pygame/blob/16a8d3ea36ff09fc1c1d91c6492219a4bef944db/src_c/_camera.c#L1841-L1851

and it looks like both turn it into `char *dev_name = NULL;` so I think str type is correct. Also not optional because it comes before the pipe `|` character.

Reply:
👍